Library/Formula/flex.rb

```diff
@@ -15,6 +15,8 @@ class Flex < Formula
   depends_on "gettext"
 
   def install
+    # 2.5.38+ build fix adapted from http://patchwork.openembedded.org/patch/68029/
+    ENV["LDFLAGS"] = "#{ENV["LDFLAGS"]} -all-static"
```

Comment:
(could also spell this `ENV.append "LDFLAGS", "-all-static"` if we go this route)